src/components/modal/LinkOutModalContents.jsx

```diff
@@ -73,103 +73,169 @@ const ButtonContainer = styled.div`
+        <ButtonDescription>
+          {`The current dataset isn't viewable in MicrobeTrace as tanglegrams aren't supported`}
+        </ButtonDescription>
+      </ButtonContainer>
+    )
+  }
+
+  return (
+    <ButtonContainer key={displayName}>
+      <ButtonText href={url()} target="_blank" rel="noreferrer noopener">{displayName}</ButtonText>
+      <ButtonDescription>
+        View this data in MicrobeTrace (<a href='https://github.com/CDCgov/MicrobeTrace/wiki' target="_blank" rel="noreferrer noopener">learn more</a>).
+        {mainTreeNumTips>500 && (
+          <span>
+            {` Note that trees with over 500 tips may have trouble loading (this one has ${mainTreeNumTips}).`}
+          </span>
+        )}
+      </ButtonDescription>
+    </ButtonContainer>
+  )
+
+  function url() {       
+    /**
+     * As of 2024-04-09, the 'origin' must be nextstrain.org or next.nextstrain.org
+     * for these links to work. This means (nextstrain.org) the links coming from heroku
+     * review apps will not work.
+     */
+    const baseUrl = 'https://microbetrace.cdc.gov/MicrobeTrace';
+    return `${baseUrl}?url=${encodeURIComponent(`${origin}${pathname}`)}`
+  }
+}
+
+
+function NextcladeLinkOut() {
+  const displayName = 'nextclade';
+  const {pathname, origin} = clientDetails();
+  const {showTreeToo} = useSelector((state) => state.controls)
+  const {mainTreeNumTips, rootSequence} = useSelector((state) => state.metadata);
+
+  // All datasets which have a root-sequence (either in-line or sidecar) can theoretically work as Nextclade
+  // datasets. See <https://github.com/nextstrain/nextclade/pull/1455> for more thorough discussion here.
+  // Excessively big trees may be problematic (as Nextclade was designed around smaller reference trees), but
+  // exactly what the threshold is isn't known. Here I use a rather ad-hoc tip-count threshold:
+  const largeTreeWarning = mainTreeNumTips > 4000;
+
+  if (
+    showTreeToo || // Tanglegrams won't work (surprise surprise!)
+    !rootSequence  // Root sequence is required for Nextclade
+  ) {
+    return (
+      <ButtonContainer key={displayName}>
+        <InactiveButton>{displayName}</InactiveButton>
+        <ButtonDescription>
+          {`The current dataset isn't viewable in Nextclade as ${
+            showTreeToo ?
+              "tanglegrams aren't supported." :
+              "this dataset doesn't have a root-sequence (either within the main JSON or as a sidecar JSON)."
+          }`}
+        </ButtonDescription>
+      </ButtonContainer>
+    )
+  }
+
+  const url = `https://clades.nextstrain.org?dataset-json-url=${encodeURIComponent(`${origin}${pathname}`)}`
+  return (
+    <ButtonContainer key={displayName}>
+      <ButtonText href={url} target="_blank" rel="noreferrer noopener">{displayName}</ButtonText>
+      <ButtonDescription>
+        View this data in Nextclade (<a href='https://clades.nextstrain.org' target="_blank" rel="noreferrer noopener">learn more</a>).
```

Comment:
This isn't the right language. It should be something more descriptive like

"Use this tree as a nextclade reference dataset which allows you to add new sequences (via drag-and-drop) and see them placed on the tree. Alignment and QC will use default parameters and it may be better to use a curated Nextclade reference dataset for this pathogen [1]. Visit [clades.nextstrain.org](https://clades.nextstrain.org) to learn more about Nextclade or read [these docs](???) for more details about using Auspice datasets in Nextclade"

Copy edits welcome!

[1] We could examine the (currently unparsed) `extensions` field in the Auspice JSON for any nextclade data

Reply:
cc @ivan-aksamentov

Reply:
LGTM.

(Perhaps instead of the longer explanation, we could link to Nextclade docs? The nuances should be explained there.)

Reply:
Thanks - settled on a terser version of the above description with links out to docs